pkg/broker/broker.go

```diff
@@ -1059,13 +1059,6 @@ func (a AnsibleBroker) Unbind(
 	} else {
 		log.Warning("Broker configured to *NOT* launch and run APB unbind")
 	}
-
-	err = cleanupUnbind(&bindInstance, &serviceInstance, bindExtCreds, a.dao)
```

Comment:
Why is this gone? It was part of the unbind_subscriber before, what is doing this now?

Reply:
I see it was moved to `cleanupAfterUnbind` in the `job_state_subscriber`.